docs/source/configuration/logging.rst

```diff
@@ -19,7 +19,9 @@ For more information, see :ref:`enabling-cassandra-tracing`.
 Debug Logging for Multiple Timestamp Services Error
 ===================================================
 
-From version 0.22, it is recommended that you send logging related to the timestamp service to a separate appender.
+This logging is available from version 0.22.0.  It is intended for targeted use only 
+(read: when you have consistent reproduction) as it is particularly verbose. 
```

Comment:
in this case, we probably ought to log at DEBUG instead of INFO for the ones that are still at such a high level.